mapproxy/util/ext/serving.py

```diff
@@ -233,8 +253,11 @@ def send_response(self, code, message=None):
         self.log_request(code)
         if message is None:
             message = code in self.responses and self.responses[code][0] or ''
-        if self.request_version != 'HTTP/0.9':
-            hdr = "%s %d %s\r\n" % (self.protocol_version, code, message)
+        if self.request_version == 'HTTP/0.9':
```

Comment:
this `if` does not make sense, i think we need this to read `!=` instead of `==`

Reply:
yes, I had reorganized the code, and left this typo